kornia/sensor/camera/camera.py

```diff
@@ -0,0 +1,320 @@
+# adapted from: https://github.com/strasdat/Sophus/blob/sophus2/cpp/sophus/sensor/camera_model.h
+from __future__ import annotations
+
```

Comment:
or maybe rename this file to `camera_model.py` so that we reduce also levels in the sub module hierarchy ?
same for the other sensor stuff